create-dmg

```diff
@@ -380,20 +399,19 @@ hdiutil resize ${HDIUTIL_VERBOSITY} -size ${DISK_IMAGE_SIZE}m "${DMG_TEMP_NAME}"
 
 # Mount the new DMG
 
-MOUNT_DIR="/Volumes/${VOLUME_NAME}"
-
-# Unmount leftover dmg if it was mounted previously (e.g. developer mounted dmg, installed app and forgot to unmount it)
-if [[ -d "${MOUNT_DIR}" ]]; then
-	echo "Unmounting old disk image from $MOUNT_DIR..."
-	DEV_NAME=$(hdiutil info | grep -E --color=never '^/dev/' | sed 1q | awk '{print $1}')
-	hdiutil_detach_retry "${DEV_NAME}"
-fi
-
 echo "Mounting disk image..."
 
-echo "Mount directory: $MOUNT_DIR"
-DEV_NAME=$(hdiutil attach -readwrite -noverify -noautoopen "${DMG_TEMP_NAME}" | grep -E --color=never '^/dev/' | sed 1q | awk '{print $1}')
+DEV_NAME=$(hdiutil attach -mountrandom /Volumes -readwrite -noverify -noautoopen -nobrowse "${DMG_TEMP_NAME}" | grep -E --color=never '^/dev/' | sed 1q | awk '{print $1}')
```

Comment:
@mbarbero this line fails on sandboxed systems (such as brew 🍻).
Specifically this part: `-mountrandom`

I think removing this one defeats the purpose of your PR, so I'm going to disable this argument only if sandboxed is enabled. Let me know if you have another solution.

Reply:
Just tried with `-mountrandom /tmp` instead of `-mountrandom /Volumes`. Do you see any inconvenient to change for all situations?

Reply:
Good catch! LGTM 👍 

Thanks for checking!